### PR TITLE
Update fix-issues-with-printers-scanners-and-lob-applications-that-se…

### PR DESCRIPTION
**Pull request by arindamthokder**
…nd-email-using-off.md

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 1]
```diff
@@ -167,6 +167,10 @@ This can be caused by a number of issues.
 
 Network or ISP changes might change your static IP address. This results in your connector not identifying and relaying your messages to external recipients. Update your connector and your SPF record with the new IP address. Follow the steps in [Option 3: Configure a connector to send mail using Microsoft 365 or Office 365 SMTP relay](how-to-set-up-a-multifunction-device-or-application-to-send-email-using-microsoft-365-or-office-365.md#option-3-configure-a-connector-to-send-mail-using-microsoft-365-or-office-365-smtp-relay) to edit your existing connector settings.
 
+### 5.7.64 TenantAttribution; Relay Access Denied or 4.4.62 Mail sent to the wrong Office 365 region
+
+This error indicates that email sent from your application or device is not being correctly [attributed](https://techcommunity.microsoft.com/t5/exchange-team-blog/office-365-message-attribution/ba-p/749143) to your tenant. A common cause of this issue is change in your dedicated and static IP address or change in the certificate used by your application or device. Update the inbound connector with the new IP address or new Certificate information.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:172]:
```suggestion
This error indicates that email sent from your application or device is not correctly [attributed](https://techcommunity.microsoft.com/t5/exchange-team-blog/office-365-message-attribution/ba-p/749143) to your tenant. A common cause of this issue is a change in your dedicated and static IP address or a change in the certificate used by your application or device. Update the inbound connector with the new IP address or new certificate information. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arindamthokder, kindly commit my suggested changes if you agree with them. Many thanks.

**Comment by arindamthokder** [comment]
> Copy review - @arindamthokder, kindly commit my suggested changes if you agree with them. Many thanks.

done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**